Abubakarharuna10.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "@Abubakarharuna10"
+project: "Artificial Rescuer"
+---
+
+# Project Name
+Artificial Rescue
+
+## Summary
+
+With the increasing threat of climate change, greater populations and consequently, more extreme environmental events
+the need for better emergency rescue services is paramount
+As time moves forward, natural disasters will continue to wreak havoc on cities and counties around the world, 
+causing many people to die. Bolstering the power of search and rescue teams with autonomous UAVs that can identify people
+in disaster relief zones could significantly reduce the impact tornadoes, earthquakes, tsunamis, floods, etc...
+My solution to strengthening search and rescue operations is to outfit an autonomous unmanned aerial vehicle (UAV) 
+with a computer vision system that detects the location of people as the vehicle flies over ground. The powerful neural-network capabilities
+of the Jetson Nano Dev Kit will enable fast computer vision algorithms to achieve this task. Joseph Redmon's YOLOv3 algorithm will be used 
+to do the actual object-detection (people) in the camera's view While the UAV is flying a waypoint mission using ArduPilot, PX4, or any other autonomous flight control stack, the absolute location of people in the camera view can be calculated based on the altitude, orientation, and GPS location of the UAV.
+
+## Plan
+
+I will use  NVIDIA Jetson Nano Developer Kit to run computer vision algorithms to detect people as the vehicle flies overhead along with an Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter that will allow me to connect to the Jetson Nano and monitor processes while the vehicle is in a flight setting then the CanaKit 5V 2.5A Power Supply will provide a stable power source while setting up the Jetson Nano and the Raspberry Pi Camera Module V2 will be mounted to the underside of the vehicle to capture video of the ground below then Anker PowerCore 5000 This power pack was selected for its 2A output and compact size. then the PX4 Pixhawk on an autonomous drone or airplane will communicate with the Jetson Nano over a wired MAVLink connection.
```

Comment:
```suggestion
I will use  NVIDIA Jetson Nano Developer Kit to run computer vision algorithms to detect people as the vehicle flies overhead. I will use an Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter that will allow me to connect to the Jetson Nano and monitor processes while the vehicle is in a flight setting. The CanaKit 5V 2.5A Power Supply will provide a stable power source while setting up the Jetson Nane. The Raspberry Pi Camera Module V2 will be mounted to the underside of the vehicle to capture video of the ground below. As forthe Anker PowerCore 5000, this power pack was selected for its 2A output and compact size. Then, the PX4 Pixhawk on an autonomous drone or airplane will communicate with the Jetson Nano over a wired MAVLink connection.
```

Suggestion: I added punctuation to make it easier to read.

Reply:
Sure Thanks for your suggestion!